tests/TestCase/View/Helper/DateRangesHelperTest.php

```diff
@@ -0,0 +1,252 @@
+<?php
+namespace Chialab\FrontendKit\Test\TestCase\View\Helper;
+
+use BEdita\Core\Model\Entity\DateRange;
+use BEdita\Core\Model\Entity\ObjectEntity;
+use Cake\I18n\FrozenTime;
+use Cake\I18n\I18n;
+use Cake\TestSuite\TestCase;
+use Cake\View\View;
+use Chialab\FrontendKit\View\Helper\DateRangesHelper;
+
+/**
+ * {@see \Chialab\FrontendKit\View\Helper\DateRangesHelper} Test Case
+ *
+ * @coversDefaultClass \Chialab\FrontendKit\View\Helper\DateRangesHelper
+ */
+class DateRangesHelperTest extends TestCase
+{
+    public $fixtures = [
+        'plugin.BEdita/Core.ObjectTypes',
+        'plugin.BEdita/Core.PropertyTypes',
+        'plugin.Chialab/FrontendKit.Properties',
+        'plugin.Chialab/FrontendKit.Relations',
+        'plugin.Chialab/FrontendKit.RelationTypes',
+        'plugin.Chialab/FrontendKit.Objects',
+        'plugin.Chialab/FrontendKit.Users',
+        'plugin.Chialab/FrontendKit.ObjectRelations',
+        'plugin.BEdita/Core.Categories',
+        'plugin.BEdita/Core.ObjectCategories',
+        'plugin.BEdita/Core.Tags',
+        'plugin.BEdita/Core.ObjectTags',
+    ];
+
+    /**
+     * Test subject
+     *
+     * @var \Chialab\FrontendKit\View\Helper\DateRangesHelper
+     */
+    protected DateRangesHelper $DateRanges;
+
+    /** @inheritDoc */
+    public function setUp()
```

Comment:
To increase compatibility with newer PHP Unit versions:

```suggestion
    public function setUp(): void
```